### ISSUE TITLE
MapTool is registered as the default handler for *all* zip files

### ISSUE THREAD

**Issue by duckinator** [RPTools/maptool#2432]
**Describe the bug**
After #1856, MapTool is registered to open all zip files.

My understanding is it is related by the first line of [`packages/associations.properties`](https://github.com/RPTools/maptool/blob/98cdf98188d2eb6de7724199af7af2c53b9a9d9e/package/associations.properties):

```
mime-type=application/zip
extension=cmpgn
description=MapTool campaign save file
```

I believe, in theory, you could just remove the `mime-type=application/zip` line and it would be fine.

**To Reproduce**
1. Install MapTool
2. Open a zip file. (even with a .zip extension)

**Expected behavior**

I expected .zip files to open with something other than MapTool.

**Screenshots**

**MapTool Info**
- Version: 1.8.3
- Install: New

**Desktop (please complete the following information):**
 - OS: Windows 10, Fedora 32 (Linux), Manjaro (Linux)

---

cc @kayila because we talked about this a bit

**Comment by Azhrei** [comment]
Agreed.  According to https://docs.oracle.com/en/java/javase/14/jpackage/support-application-features.html#GUID-324F3A7B-409A-426D-AFB1-E4540049D13E it should be possible to remove the MIME a type completely (a or it should be changed to `application/maptool` or similar).

(If we're going to modify the build process, we should also add `--linux-shortcut` as Debian-based Linux users have indicated they're not getting a desktop shortcut created when they install MapTool.)

**Comment by Phergus** [comment]
@duckinator 
> I believe, in theory, you could just remove the mime-type=application/zip line and it would be fine.

Was thinking I'd just change it to `mime-type=application/maptool` to be in line with other application files.

@Azhrei 
> (If we're going to modify the build process, we should also add `--linux-shortcut` as Debian-based Linux users have indicated they're not getting a desktop shortcut created when they install MapTool.)

It already has it as well as the menu group designation. 
```
installerOptions += [
	'--linux-menu-group', 'Game',
	'--linux-shortcut'
]
```

**Comment by perception101** [comment]
Caused quite a big of havoc at my end, triggering Firefox to start saving zip files as file.zip.cmpgn and always try to open them in MapTool.  Worth include a fix to 1.8.4 to back-out the incorrect association too?
